invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/components/DocumentMetadata/DocumentMetadataTabs.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { Divider, Tab } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { DocumentRelations, DocumentInfo } from './components';
+import { DocumentTableOfContent } from './components/DocumentTableOfContent';
+import { DocumentConference } from './components/DocumentConference';
+
+export default class DocumentMetadataTabs extends Component {
```

Comment:
Coming to @zzacharo comment from the previous pr, this is a Stateless component used only for presentation purposes, should we make the functions instead of classes?

Reply:
to note: as we discusses in standup we will go for class components